core/cmd/local_client.go

```diff
@@ -365,7 +365,7 @@ func (cli *Client) RebroadcastTransactions(c *clipkg.Context) (err error) {
 	}
 	defer lggr.ErrorIfClosing(db, "db")
 
-	app, err := cli.AppFactory.NewApplication(cli.Config, db)
+	app, err := cli.AppFactory.NewApplication(context.Background(), cli.Config, db)
```

Comment:
nbd but is this the best we can do here?
I see there is a `context.TODO()` on 389 too.

Reply:
We don't have a context passed to CLI command handlers. Probably this deserves another story to add a SIGTERM/INT handler bound to a context. But for CLI usage I don't see a significant impact, because shutdown time is not a concern there. Do you agree?